### DISCUSSION TITLE
请问支持官网的Qwen1.5 32B awq模型吗？

### DISCUSSION THREAD

**Discussion by lai-serena** [InternLM/lmdeploy#2363] [Q&A] [answered]
请问支持官网的Qwen1.5 32B awq模型吗？
但是会出现这个错误
![image](https://github.com/user-attachments/assets/2b44a99b-d463-45a3-a443-4799ec781d85)

**Comment by lvhan028** [comment, marked as answer]
```
  "quantization_config": {
    "bits": 4,
    "group_size": 32,
    "modules_to_not_convert": null,
    "quant_method": "awq",
    "version": "gemm",
    "zero_point": true
  },
```
the quant_config of qwen1.5-32b is listed above. 
The group_size is 32, which is not supported by lmdeploy. 
Currently, lmdeploy only supports group_size 128
